tensorflow/python/ops/sparse_ops.py

```diff
@@ -2674,24 +2674,15 @@ def sparse_softmax(sp_input, name=None):
 
   Example:
 
-  ```python
-  # First batch:
-  # [?   e.]
-  # [1.  ? ]
-  # Second batch:
-  # [e   ? ]
-  # [e   e ]
-  shape = [2, 2, 2]  # 3-D SparseTensor
-  values = np.asarray([[[0., np.e], [1., 0.]], [[np.e, 0.], [np.e, np.e]]])
-  indices = np.vstack(np.where(values)).astype(np.int64).T
-
-  result = tf.sparse.softmax(tf.sparse.SparseTensor(indices, values, shape))
-  # ...returning a 3-D SparseTensor, equivalent to:
-  # [?   1.]     [1    ?]
-  # [1.  ? ] and [.5  .5]
-  # where ? means implicitly zero.
-  ```
-
+  >>> shape = [2, 2, 2]  # 3-D SparseTensor
+  >>> values = np.asarray([[[0., np.e], [1., 0.]], [[np.e, 0.], [np.e, np.e]]])
+  >>> indices = np.vstack(np.where(values)).astype(np.int64).T
+  <tf.Tensor(
+   [[[0.  1. ]
+   [1.  0. ]]
+   [[1.  0. ]
+   [0.5 0.5]]], shape=(2, 2, 2), dtype=float64)>
```

Comment:
I don't think this is the right output. Can you make sure you run the doctest?

Reply:
Hi @mihaimaruseac! 
I have changed the code from first cell to second cell as shown in this [gist](https://colab.research.google.com/gist/sushreebarsa/792f4b3f9df002ee48f2e7d98d7a6e23/untitled60.ipynb). Please confirm and let me know if further modifications are required. Thank you!

Reply:
Please run the doctest. You will see it failing, because `x = expr` does not output.

Also, this doctest does not seem to use the TF API at all. Please update.